build/common/run-e2e-tests.yml

```diff
@@ -6,7 +6,7 @@ steps:
     downloadPath: '$(System.ArtifactsDirectory)'
     artifactName: 'IntegrationTests'
 
-- script: dotnet test "Microsoft.Health.Dicom.Web.Tests.E2E.dll" --filter "FullyQualifiedName!~Microsoft.Health.Dicom.Web.Tests.E2E.Rest.DataPartitionEnabledTests" --logger trx --results-directory "$(Agent.TempDirectory)/TestResults"
+- script: dotnet test "Microsoft.Health.Dicom.Web.Tests.E2E.dll" --filter Category=bvt --logger trx --results-directory "$(Agent.TempDirectory)/TestResults"
```

Comment:
I would think that we'd want to run all the tests except the bvt-dp here. Where would be testing something like this now? https://github.com/microsoft/dicom-server/blob/1f966e64e4441c6d22135a26393bd85c4ca7381a/test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ChangeFeedTests.cs#L84

Reply:
I agree, this shouldn't be bvt. bvt or bvt -dp applies to paas ci e2e tests.

Reply:
Reverted this change.